docs/antrea-network-policy.md

```diff
@@ -500,9 +513,14 @@ the policies that may affect their workloads.
 - There is a soft limit of 20 on the maximum number of Tier resources that are
   supported. But for optimal performance, it is recommended that the number of
   Tiers in a cluster be less than or equal to 10.
-- The v1alpha1 Policy CRDs support up to 10000 unique priority at policy level.
-  In order to reduce the churn in the agent, it is recommended to set the
+- In order to reduce the churn in the agent, it is recommended to set the policy
   priority within the range 1.0 to 100.0.
+- The v1alpha1 Policy CRDs support up to 10,000 unique priorities at policy level,
+  and up to 50,000 unique priorities at rule level, across all tiers except for
+  the "baseline" tier. For any two policy rules, their rule level priorities are only
+  considered equal if they share the same tier, and have the same policy priority
+  as well as rule priority.
+- For the "baseline" tier, the max supported unique priorities (at rule level）is 150.
```

Comment:
I feel like this could be improved to include more information (rule priorities are assigned at the Node level) and make it clearer, but that can be done in a future PR

Reply:
I actually think it is fine as is, at least for now. Introducing Node level max priorities feels a bit too much implementation detail to me... also for those who actually try to push the scale to the limit and create the max allowed number of priorities for Pods on one Node, and totally different, also max allowed number of priorities for Pods on another Node, issue may occur as soon as some Pods go down and get schedule on another Node... This is a very extreme example, but I guess my point is 150 priorities for the baseline tier across cluster and 50,000 for the rest tiers are more than sufficient for consumption, and this soft limit also leaves us some buffer.

Reply:
Makes sense. We can revisit if we get some user requests for clarifications